src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const next = (...args) => {
+    return (x) => {
```

Comment:
why do you need this additional function?
if your "next" function will take x and you return not a call to "next" function with arguments but just a "next" function, it will work and you will avoid the extra function, if you have any questions feel free to ask this in fe_chat2